docs/release.md

```diff
@@ -0,0 +1,54 @@
+# Expected workflow for the development to release process
+
+## Branching strategy
+
+- Checkout the `development` branch after cloning (or forking) the repository
+- Create an appropriately named 'feature', 'hotfix', 'patch', etc branch
+
+## Commits
+
+To easily facilitate CHANGELOG generation, automatic version bumps (using [SEMVER](https://semver.org) - X.Y.Z) and version tagging, commit messages will HAVE to follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) practice, which is used in combination with the GitHub Workflow Action, [TriPSs/conventional-changelog-action](https://github.com/TriPSs/conventional-changelog-action).
```

Comment:
```suggestion
To easily facilitate CHANGELOG generation, automatic version bumps (using [SEMVER](https://semver.org) - X.Y.Z) and version tagging, commit messages MUST follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) practice, which is used in combination with the GitHub Workflow Action, [TriPSs/conventional-changelog-action](https://github.com/TriPSs/conventional-changelog-action).
```

The future tense feels appropriate for a proposal, but it won't fit well when we merge this and the document is active policy. Using "MUST" eliminates the aspirational tone and aligns with the RFC2119 language used elsewhere in the document.